### ISSUE TITLE
VoiceOver and keybaord support for platform table filter does not work properly

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12770]
#### Is this a bug, enhancement, or feature request?

bug

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

angular: 18.2.4
fundamental: 0.53.1-rc.23

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

Create platform table with filter (it can be done based on [this example](https://sap.github.io/fundamental-ngx/#/platform/table/basic#initial-state) .

1. Try to set some table filter using only keyboard. List of filterable columns should appear ('price', 'status', 'status color'). It is not possible to use only keyboard (space or enter key) to go inside 'price' or 'status' or 'status color' and set filter values.

2. Try to set some table filter using only VoiceOver (MacOs version 14.7). After selecting first filter property ('price'), the focus on filter dialog is lost and moved to main window.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.

Accessibility expert comment: _there is a trap for a user- it is not possible to go further with the reading because of the loop from filter to main window_
